rdrf/rdrf/urls.py

```diff
@@ -312,8 +313,9 @@ def proms_only(url_pattern):
             name='registration_activation_complete'),
 
     re_path(r'^activate/(?P<activation_key>\w+)/?$',
-            ActivationView.as_view(),
+            PatientActivationView.as_view(),
             name='registration_activate'),
+    re_path(r'^resend-activation/?$', ResendActivationView.as_view(), name='registration_resend_activation'),
```

Comment:
Worth adding? I'm not sure if there are security concerns with allowing patients to initiate re-activation.

Reply:
I think it's fine to have this as a backup but only admins/clinicians should be able to resend the activation mail for a patient.